_includes/mandatory-reporters.html

```diff
@@ -0,0 +1,22 @@
+<section>
+    <header>
+        <h3 id="mandatory-reporters">Mandatory Reporters</h3>
+        <div class="buttons"><a href="#alternative-resources">Back to the top</a></div>
+    </header>
+
+    <p>
+        Certain resources' representatives are what're called <em>mandatory reporters</em>. Mandatory reporters are required to report certain things to the police. For example, in most states and US territories, mandatory reporters are obliged to report actual or suspected child abuse or neglect to the state.
```

Comment:
Just some wording suggestions
```suggestion
        Certain resources' representatives are <em>mandatory reporters</em>. Mandatory reporters are required to report certain things (typically situations involving potential illegalities) to the police. For example, in most states and US territories, mandatory reporters are obliged to report actual or suspected child abuse or neglect to the state.
```